tests/CMakeLists.txt

```diff
@@ -92,10 +93,12 @@ gu_test(run_degrib2_tests)
 gu_test(run_grbindex_tests)
 gu_test(run_grb2index_tests)
 
-# This test depends on the g2c_compare utility, which may not be
-# present.
-if (G2C_COMPARE)
+option(G2C_COMPARE OFF)
```

Comment:
Can you put this option in the top-level cmake file, with all the other build options?